src/fondue/Fondue.js

```diff
@@ -277,6 +277,33 @@ export default class Fondue {
 		return tables.filter((table) => colorTables.includes(table));
 	}
 
+	get colorPalettes() {
+		const cpal = this._font.opentype.tables.CPAL;
+		if (!cpal) return [];
+
+		// Create padded hex value for color string
+		const hex = (d) => Number(d).toString(16).padStart(2, "0");
+
+		// Convert color records to RGBA hex strings
+		// and group them per palette
+		return cpal.colorRecords.reduce((colors, clr, index) => {
```

Comment:
Could you perhaps add a small remark here that `cpal.colorRecords` is a flat array, which you're converting into an array of arrays? It's not immediately clear from the code, but I'm not sure how to simplify it either so a small comment might help understanding this code.